proto/sentry_protos/snuba/v1/endpoint_trace_item_attributes.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+
+//A request for "which queryable attributes are available for these projects between these dates"
+// - used for things like autocompletion
+// EAP does not make a distinction between data stored as columns on a table vs attributes stored
+// dynamically (e.g. tags, contexts, etc). To the user of the interface, they are the same
+message TraceItemAttributesRequest {
```

Comment:
I think this should have the `value_substring_match` field as well, for auto complete.

Reply:
Done